csvcubed/csvcubed/models/rdf/namespaces.py

```diff
@@ -0,0 +1,51 @@
+"""
+Namespaces
+----------
```

Comment:
These Namespaces should be imported form csvcubed-models.rdf.namespaces (the modules library exists to reference external models)

Reply:
I don't think this is resolved. Have a chat with @canwaf about this one.

Reply:
the import had been updated, this was inadvertently left in, file is now deleted.